index.js

```diff
@@ -6,12 +6,39 @@ module.exports = {
   extends: ['airbnb'],
   plugins: ['chai-friendly'],
   rules: {
-    'no-unused-expressions': 0,
-    'chai-friendly/no-unused-expressions': 2,
+    'no-unused-expressions': 'off',
+    'chai-friendly/no-unused-expressions': 'error',
     'linebreak-style': ['error', 'unix'],
     'no-param-reassign': ['error', { props: false }],
     'func-style': ['error', 'declaration', { allowArrowFunctions: true }],
     'no-use-before-define': ['error', { functions: false }],
-    'consistent-return': 0,
+    'no-unused-vars': ['error', {'vars': 'all', 'args': 'none'}],
+    'consistent-return': 'off',
+    'arrow-body-style': ['warn', 'as-needed', {
+      requireReturnForObjectLiteral: false,
+    }],
+    'prefer-rest-params': 'warn',
+    'prefer-destructuring': ['warn', {
+      VariableDeclarator: {
+        array: false,
+        object: true,
+      },
+      AssignmentExpression: {
+        array: true,
+        object: true,
+      },
+    }, {
+      enforceForRenamedProperties: false,
+    }],
+
+    'no-underscore-dangle': 'off',
+    'array-element-newline': 'off',
+    'function-paren-newline': ['warn', 'multiline'],
+    'newline-per-chained-call': ['warn', { ignoreChainWithDepth: 4 }],
+    'no-inline-comments': 'off',
+    'object-curly-newline': ['warn', {
```

Comment:
The whole reason of this file is too warn about styling, not making it errorous, so that styling becomes a guideline rather than an enforcement. I'm eager to fix a warning or to disable it on per-cases